apps/easypid/src/agent/index.ts

```diff
@@ -4,9 +4,9 @@ import {
   type ParadymAppAgent,
   isEasyPIDAgent,
   isParadymAgent,
-  useAgent,
 } from '@package/agent'
 import { useSecureUnlock as _useSecureUnlock } from '@package/secure-store/secureUnlock'
+import { useAgent } from '@paradym/wallet-sdk/src/providers/AgentProvider'
```

Comment:
I updated all the paths in the expo update pr to not use `/src` anymore. By adding a tsconfig and metro/babel mapping.

I think for consistency would be great to do it here as well. (also, but mabye for future, it might be nice to look at export mapping in package json so we can have multiple export for different features, e.g. `@paradym/wallet-sdk/providers` , etc..)